app/assets/javascripts/modules/moj.BookToNomis.js

```diff
@@ -24,10 +22,10 @@
     render: function () {
       var $el = $('#'+this.$conditionalEl);
```

Comment:
I hadn't noticed the $ (I probably copied and pasted from another file), I will update the variable name